app/services/sync_provider_from_find.rb

```diff
@@ -37,6 +37,12 @@ def self.update_provider_name(provider, provider_name)
     provider.save!
   end
 
+  def self.update_provider(provider, find_provider)
+    provider.region_code = find_provider.region_code.strip if find_provider.region_code
```

Comment:
Will this crash when calling `strip` on `nil`?

Reply:
I don't think it will due to the `if find_provider.region_code`